debian/DEBIAN/control

```diff
@@ -0,0 +1,12 @@
+Source: snowflake
```

Comment:
Do not put a `/debian` folder in release tarballs, so if this is accepted tagging realeses would be bad.  The reason is Debian proper cannot package non-native, that's this project, cleanly...  The source diff looks either ugly or incomplete.

Instead put these files in a staging folder and have build scripts copy or link them into place.  Debian has no standard for doing this, but it is a rule not to package projects with a clashing folder.